db/models.js

```diff
@@ -81,10 +77,16 @@ const UserLocal = sequelize.define('userlocal', {
     id: {type: Sequelize.INTEGER, primaryKey: true, autoIncrement: true},
     username: {type: Sequelize.STRING, unique: true},
     password: Sequelize.STRING
+    type: Sequelize.STRING
 });
 
+// Table to tsore common proerties of students and teachers
 const Users = sequelize.define('user', {
     id: {type: Sequelize.INTEGER, primaryKey: true, autoIncrement: true}
+    name: Sequelize.STRING,
+    roll: {type: Sequelize.STRING, unique: true},
+    email: {type: Sequelize.STRING, isEmail: true},
+    type: Sequelize.STRING
```

Comment:
we don't need type here either. we are using relationships in DB (hasOne - belongsTo) to handle that